sim/warrior/charge.go

```diff
@@ -0,0 +1,33 @@
+package warrior
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+)
+
+func (warrior *Warrior) RegisterCharge() {
```

Comment:
Can charge be used in combat and with no minimum range now?

Reply:
Juggernaut makes Charge and Intercept share a CD.  I did not add intercept as there is no DPS use currently.